docs/getting-started/overview.rst

```diff
@@ -1,6 +1,45 @@
-.. include:: /stub-topic.txt
+Overview of different editions of .NET
+======================================
+
+With the advent of .NET Core and .NET Native, the layout of the .NET ecosystem has changed significantly. Where before there was a (somewhat) simple mapping between platforms and .NET Framework versions, today we have a lot of 
```

Comment:
I think you mean "introduction" more than "advent" ...

We should avoid thinking of .NET Native as a different edition of .NET and more as a build optimization for .NET Core.

I'd prefer something different than "Overview of different editions of .NET". How about "Overview of .NET implementations". I'd then include Mono and IL2CPP in that too.

Try to avoid the historical reference. When I write docs, I'm tempted to do that, too. I think it's much better if you describe how things are. This is useful for two reasons: not everyone is going to have/benefit from historical contents and the historical context will be less useful over time.